src/electron/platform/android/appium-service-configurator.ts

```diff
@@ -82,9 +82,10 @@ export class AppiumServiceConfigurator implements AndroidServiceConfigurator {
         await this.adb.uninstallApk(servicePackageName);
     };
 
-    public setTcpForwarding = async (deviceId: string): Promise<void> => {
+    public setTcpForwarding = async (deviceId: string): Promise<number> => {
         this.adb.setDeviceId(deviceId);
         await this.adb.forwardPort(defaultAdbPortNumber, defaultAdbPortNumber);
```

Comment:
Even with a dynamic port, only the first port can change. The second port is what the service listens on and is non-configurable

Reply:
Agreed, that's why it's returning `Promise<number>` rather than `Promise<{hostPort: number, devicePort: number}>`